sorc/link_workflow.sh

```diff
@@ -91,6 +91,7 @@ for dir in aer \
             cpl \
             datm \
             gldas \
+            gsi \
```

Comment:
How do we handle the case that a developer does NOT want to use the staged GSI fix files?   This question actually applies to all the staged fix files.  One answer is that we assume the developer knows what (s)he wants to do and manually makes the required changes.

Reply:
Since all that is being done is creating symlinks, it is a simple matter to replace one of them with a location of your choice manually afterwards.

Reply:
Agreed.  It is simple to replace one link with another.  Does the g-w wiki capture this detail?   Not everyone understands what these scripts do or how they work.